packages/govern-server/src/index.ts

```diff
@@ -1,11 +1,17 @@
-import { graphqlHttpPort } from './env'
+import env from './env'
 import startGraphql from './api-graphql'
+import GovernData from './core/data'
 
 async function main() {
-  const _graphqlHttpPort = graphqlHttpPort()
-  const { url } = await startGraphql({ httpPort: _graphqlHttpPort })
+  const govern = new GovernData({ network: 4 })
+
+  const { url: graphqlUrl } = await startGraphql({
+    govern,
+    httpPort: env.graphqlHttpPort,
+  })
+
   console.log()
-  console.log(`GraphQL API listening on ${url}`)
+  console.log(`GraphQL API listening on ${graphqlUrl}`)
```

Comment:
Just a side-note: Is it planned to use an advanced logging mechanism in the final product as for example the Winston logger?

Reply:
Definitely yes. I put having “complete logs” in the [phase 4](https://github.com/aragon/govern/issues/49), but we could start having some logging mechanism before that.

Reply:
Noice! :)